src/main/resources/assets/amunra/lang/en_US.lang

```diff
@@ -37,13 +37,13 @@ tile.smoothbasalt.slab.name=Smooth Basalt Slab
 tile.smoothbasalt.stairs.name=Smooth Basalt Stairs
 tile.blockUranium.name=Block of Uranium
 tile.mothershipController.name=Mothership Navigation Console
-tile.mothershipController.description=This block is used to control destination of Motherships.
+tile.mothershipController.description=This block is used to control destination of Motherships, and to change permissions for it.
 tile.mothershipEngineRocket.name=Rocket Engine
 tile.mothershipEngineRocketJet.name=Rocket Engine Jet
 tile.mothershipEngineRocketBooster.name=Rocket Engine Base
 tile.mothershipEngineRocket.description=Place one or more engine base blocks in a row, and one jet block on one side to create a mothership engine, to allow it to move
 tile.mothershipSettings.name=Mothership Configuration Terminal
-tile.mothershipSettings.description=Customize Mothership's look on map and name
+tile.mothershipSettings.description=Customize Mothership's look on map and name. for organisation.
```

Comment:
It was meant to be a comma. And, i meant for organization that, if you have a lot of MS, it would be more organized to have use icons ^^